data/mods/Aftershock/mobs/zombies.json

```
@@ -518,5 +518,164 @@
+    "default_faction": "moxie",
+    "bodytype": "human",
+    "species": [ "MOXIE" ],
+    "diff": 10,
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 270,
+    "speed": 120,
+    "material": [ "flesh" ],
+    "symbol": "E",
+    "color": "i_red",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_cut": 4,
+    "dodge": 4,
+    "vision_day": 10,
+    "regenerates": 15,
+    "armor_bullet": 30,
+    "armor_bash": 15,
+    "armor_cut": 20,
+    "harvest": "alien_moxphore",
+    "special_attacks": [
+      [ "LONGSWIPE", 20 ],
+      [ "GRAB", 20 ],
+      {
+        "id": "scratch",
+        "cooldown": 10,
+        "accuracy": 6,
+        "damage_max_instance": [
+          { "damage_type": "stab", "amount": 8, "armor_multiplier": 0.3 },
+          { "damage_type": "bash", "amount": 16, "armor_multiplier": 0.3 }
+        ]
+      }
+    ],
+    "death_function": [ "NORMAL" ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "GOODHEARING",
+      "KEENNOSE",
+      "SWIMS",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "DESTROYS",
+      "POISON",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "PUSH_VEH",
+      "FILTHY"
+    ]
+  },
+  {
+    "id": "mon_moxie_bloodhound",
+    "type": "MONSTER",
+    "name": { "str": "MoxPhore Bloodhound" },
+    "description": "An arachnid humanoid shape, twisted into a canine-like form.  It scuttles across the ground, looking blankly with vestigal eyes as it sniffs the air.",
+    "default_faction": "moxie",
+    "bodytype": "dog",
+    "species": [ "MOXIE" ],
+    "diff": 10,
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 70,
+    "speed": 120,
+    "material": [ "flesh" ],
+    "symbol": "Q",
+    "color": "red",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 2,
+    "melee_dice_sides": 2,
+    "melee_cut": 2,
+    "dodge": 2,
+    "vision_day": 2,
+    "regenerates": 7,
+    "armor_bullet": 20,
+    "armor_bash": 10,
+    "armor_cut": 10,
+    "harvest": "alien_moxphore",
+    "special_attacks": [
+      { "type": "bite", "cooldown": 5 },
+      { "id": "scratch", "damage_max_instance": [ { "damage_type": "cut", "amount": 6 } ] }
+    ],
+    "death_function": [ "NORMAL" ],
+    "flags": [
+      "STUMBLES",
+      "HEARS",
+      "GOODHEARING",
+      "KEENNOSE",
+      "SWIMS",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "FILTHY"
+    ]
+  },
+  {
+    "id": "mon_moxphore_emetic",
+    "type": "MONSTER",
+    "name": { "str": "Moxphore Emetic" },
+    "description": "An arachnid humanoid stands here, hunched over as it carries itself with its arms.  Bile and green liquid leaks out of its mandibles as it stumbles, looking nauseous.",
```

```suggestion
    "description": "The walking remains of long dead colonist, hunched  and wandering in apparent pain.   An alien plant peeks meekly from its open mouth, leaking green liquid as its host stumbles.",
```
You mentioned having mixed feelings about the justification of this one. An idea that occurred to me was to make it an symbiotic union of two degraded machines.

---

"walking remains of long dead colonist" -> "walking remains of a long-dead colonist"